src/components/route-link/route-link.tsx

```diff
@@ -36,8 +36,15 @@ export class RouteLink {
   @Prop() history: RouterHistory;
   @Prop() location: LocationSegments;
   @Prop() root: string;
+  
+  @Prop() ariaHasPopup: string;
+  @Prop() id: string;
+  @Prop() ariaPosInset: string;
+  @Prop() ariaSetSize: number;
```

Comment:
Change name to be `ariaHaspopup`, `ariaPosinset` and `ariaSetsize`.  We convert the dashes to uppercase so since aria properties only have a single dash we would want to capitalize only the first letter after the dash.

Reply:
Thanks for the review and review comments. Done with the expected changes. Please review.